parquet-hadoop/src/main/java/org/apache/parquet/hadoop/rewrite/RewriteOptions.java

```diff
@@ -350,6 +436,28 @@ public Builder indexCacheStrategy(IndexCache.CacheStrategy cacheStrategy) {
       return this;
     }
 
+    /**
+     * Set a flag weather columns from join files need to overwrite columns from input files.
+     *
+     * @param joinColumnsOverwrite
```

Comment:
Remove this line if no doc is provided

Reply:
I thought we can focus on documentation and the tests at the end after we agree on implementation :-)